docs/src/inputs/select/examples/exampleLabelSelect.jsx

```diff
@@ -0,0 +1,28 @@
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import {
+    Select,
+} from 'react-cm-ui';
+import { options } from '../constants';
+
+function ExampleLabelSelect() {
+    const [selectedValue, setOption] = useState({});
+    function onSelect(selectedOption) {
```

Comment:
NAB: Need to add an empty newline

Reply:
fixed.

Reply:
@cbrewer1689 Could you please check once, I have checked commits but doesn't seems to be fixed ?